src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -604,5 +604,5 @@ description="Why and how I migrated this website from Gatsby to Astro."/>
 </CardGrid>
 
 :::note[Have a resource to share?]
-If you found (or made!) a helpful video or blog post about converting a Gatsby site to Astro, [edit this page](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx) and add it here!
+If you found (or made!) a helpful video or blog post about converting a Gatsby site to Astro, edit this page and add it here!
```

Comment:
```suggestion
If you found (or made!) a helpful video or blog post about converting a Gatsby site to Astro, [add it to this list](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx)!
```
OK, I think this is a non-clunky way to handle this that can still be translated just fine but also point to the English page. Now, we should change all of the migration guides!